config/settings.yml

```diff
@@ -59,9 +59,11 @@ aws:
   access_key_id: 'fake-access'
   secret_access_key: 'fake-secret'
   speech_to_text:
+    # values below can be found by running `terraform output` in terraform_aws/organizations/{env}/speech_to_text
     role_arn: 'arn:aws:iam::1234567890123:role/DevelopersRole'
     base_s3_bucket: 'sul-speech-to-text-dev' # default bucket for storing speech-to-text file
-    sqs_todo_queue_url: 'https://sqs.us-west-2.amazonaws.com/queue_url' # queue to send new speech-to-text jobs
+    batch_job_queue: 'arn:aws:batch:us-west-2:1234567890123:job-queue/sul-speech-to-text' # AWS Batch queue to submit jobs to
+    batch_job_definition: 'arn:aws:batch:us-west-2:1234567890123:job-definition/sul-speech-to-text:1' # AWS Batch job definition to use for job
     sqs_done_queue_url: 'https://sqs.us-west-2.amazonaws.com/queue_url_done' # queue to receive completed speech-to-text jobs
```

Comment:
are we sure we want to check in these actual values into the repo here instead of just putting in placeholders and then having the actual values in shared_configs per environment?

Reply:
Sure we could put placeholders here. They aren't really secret, but it could be confusing.

Reply:
I've updated it with values that look less real, and also updated a test that was expecting the hard coded value.

Reply:
chiming in to observe that while the `role_arn` values look legit at a glance, they're fake too, they just have the same number of digits as a real `role_arn`.

Reply:
(i would vote for leaving them as-is, but don't feel strongly -- i have a slight preference for very realistic looking example values)